metadata.yml

```
@@ -562,3 +562,12 @@ blogs:
             email. But what if you want this email to have a pre-filled attachment?
             Let's dive into how to do this with Symfony Mailer.
         category: tech
+
+    install-php-locally:
+        title: "Installing PHP Locally (for Symfony CLI)"
+        published: 2025-07-22
+        preview: |
+            The Symfony CLI is *almost* fully batteries included, but it does require
```

Grammarly suggests it this way, accurate?
```suggestion
            The Symfony CLI is *almost* fully battery-included, but it does require
```